hs_app_netCDF/models.py

```diff
@@ -51,18 +51,24 @@ def create(cls, **kwargs):
 
         :param kwargs: the 'value' in kwargs should be a dictionary
         """
-
+        value_arg_dict = None
         if 'value' in kwargs:
+            value_arg_dict = kwargs['value']
+        elif '_value' in kwargs:
+            value_arg_dict = json.loads(kwargs['_value'])
```

Comment:
the documentation you added describes it as a "unicode string dictionary" when it is in fact JSON serialized and should be described as such. they are very similar but not the same things.